### PR TITLE
feat(cloud-masthead): adjust "Contact us" button visibility

### PR DESCRIPTION
**Pull request by huntermacd**
### Related Ticket(s)

https://jsw.ibm.com/browse/HC-1810

### Description

This diff adds a new attribute to the cloud-masthead-composite component that controls the visibility of the Contact us button. Additionally, updated the Storybook story to include a toggle so this can be seen in action.

image

Screen Shot 2021-04-22 at 2 32 12 PM

### Changelog

**New**

- n/a

**Changed**

- cloud-masthead-composite.ts
- cloud-masthead.stories.ts

**Removed**

- n/a

### CODE REVIEW

**Comment by shixiedesign** [comment]
Had a discussion with @proeung and I understand what this PR does now. Good to go through our regular code review process and no need for design review. Thanks!

File: packages/web-components/src/components/masthead/cloud/cloud-masthead-composite.ts [hunk 1]
```diff
@@ -212,7 +219,11 @@ class DDSCloudMastheadComposite extends DDSMastheadComposite {
                       `
                   )}
                 </dds-cloud-masthead-profile>
-                <dds-cloud-button-cta kind="ghost">${contactUsButton?.title}</dds-cloud-button-cta>
+                ${hasContact
+                  ? html`
+                      <dds-cloud-button-cta kind="ghost">${contactUsButton?.title}</dds-cloud-button-cta>
```
Comment (proeung) [inline packages/web-components/src/components/masthead/cloud/cloud-masthead-composite.ts:224]: We'll need to account for the `data-ibm-contact=` attribute to trigger the "Let's Chat" contact module. Here's the link to the doc that Jeff shared with us - https://www.ibm.com/standards/web/v19a-archive/components/contact-module/#call-to-actions

**Review by proeung — CHANGES_REQUESTED** [review]
@huntermacd Looks good overall! Thanks for adding the knob for changing the visibility of the "Contact us" button. 

I think the one thing that we're missing is the ability to set this button in a way that triggers the "Let's Talk" contact module on pages where it's being used.

**Review by proeung — APPROVED** [review]

**Comment by proeung** [comment]
@huntermacd Thanks for addressing my feedback and let's make sure the branch is up to date with the base branch.

@annawen1 This PR is ready for a code review.

**Review by annawen1 — APPROVED** [review]
LGTM! Thanks @huntermacd

**Review by ariellalgilmore — COMMENTED** [review]
Looks good! I noticed though this is only for desktop and just wanted to confirm that this is intended.

**Comment by ltitle** [comment]
I think we discussed removing Contact us on mobile due to screen real estate and because the lets talk icon will be in close proximity.

**Comment by proeung** [comment]
@ltitle Thanks for the confirmation! I think this PR is good for merging then!

**Pull request merged**